packages/apm/src/integrations/tracing.ts

```diff
@@ -505,6 +572,16 @@ export class Tracing implements Integration {
 
     Tracing._log('[Tracing] Adding & adjusting spans using Performance API');
 
+    // FIXME: depending on the 'op' directly is brittle.
+    if (transactionSpan.op === 'pageload') {
```

Comment:
I didn't see an obvious quick fix for this brittleness.

The problem:

- Any transaction could be called `pageload`, and that should not mean it should report the LCP;
- The "pageload" transaction could be called something else, and we'd miss the LCP;

Reply:
We could add additional things into the `data` property and check for that but tbh I think for now this is enough.